src/reader/__tests__/fixtures/markdown/html-comment-invalid-yaml.md

```diff
@@ -0,0 +1,4 @@
+<!--
```

Comment:
Do you use this fixture in tests?

Reply:
Yeah it's used by the `markdown-tests`: https://github.com/stoplightio/markdown/blob/fa2445e6504ac8addfa0e21cdf7acd51db5e2b1b/src/reader/__tests__/markdown.spec.ts

Reply:
Great, let's have it in.